### ISSUE TITLE
Traiing stoped after 100 iteration. Why?

### ISSUE THREAD

**Issue by astronaut71** [AlexeyAB/darknet#2449]
Hi

Hi I follow the tutorial How to train (to detect your custom objects). I made all needed changes and steps. Im using Ubuntu 16.04 GPU GTX 1060, CUDA = 9.0.176 and CuDNN for CUDA 9.
I have only one class. At the beginning just wanted to have some results so (training is only 400 images) ,my batch=64,subdivisions=64 ,classes=1,filter=18

when run
`./darknet detector train /home/darknet/data/workers.data /home/darknet/cfg/yolo-obj.cfg darknet53.conv.74`

At the Iteration 99 is:

> 99: 469.803864, 848.945251 avg loss, 0.000000 rate, 20.457852 seconds, 6336 images
> Loaded: 0.000029 seconds

and 100 iteration the training stoped. 

> 100: 443.011078, 808.351807 avg loss, 0.000000 rate, 20.684246 seconds, 6400 images
> Saving weights to /home/rtc/darknet/backup//yolo-obj_last.weights
> Couldn't open file: /home//darknet/backup//yolo-obj_last.weights

I have two doubts. First, why the rate is always 0.00000? And second why the path is incorrect to save the weight , and why cant open it? 

Any help?

**Comment by astronaut71** [comment]
Was my bad. manually I created backup folder and trained again. Also in the  obj.data file the backup path should be backup = /home/darknet/backup

Now is running ok . Only why the rate is always 0? And is it normal by iteration 100 the avg loss is 850?

Thanks

**Comment by aadeshb7** [comment]
the rate can be specified in the cfg file. change the "steps" parameter so that the rate will change at the given iteration number

**Comment by AlexeyAB** [comment]
> Now is running ok . Only why the rate is always 0? And is it normal by iteration 100 the avg loss is 850? Its constantly decrease , so started from 4000 and going down to 850 by iteration =100

This is normal, at 1000 iterations learning_rate will be equal to lr from cfg-file, due to `burn_in=1000` in your cfg-file.

**Comment by astronaut71** [comment]
ok. thanks
